MAUI/Shimmer/custom-view.md

```diff
@@ -0,0 +1,735 @@
+                HeightRequest = 10,
+                WidthRequest = 160
+            };
+            
+            var boxView12 = new BoxView
+            {
+                HorizontalOptions = LayoutOptions.Start,
+                VerticalOptions = LayoutOptions.Start,
+                BackgroundColor = Color.Gray,
+                HeightRequest = 10,
+                WidthRequest = 175
+            };
+            
+            var boxView3 = new BoxView
+            {
+                BackgroundColor = Color.Gray,
+                Margin = new Thickness(0, 5, 0, 0)
+            };
+            
+            var boxView14 = new BoxView
+            {
+                HorizontalOptions = LayoutOptions.Start,
+                VerticalOptions = LayoutOptions.Start,
+                BackgroundColor = Color.Gray,
+                HeightRequest = 10,
+                WidthRequest = 150
+            };
+            
+            var boxView15 = new BoxView
+            {
+                HorizontalOptions = LayoutOptions.Start,
+                VerticalOptions = LayoutOptions.Start,
+                BackgroundColor = Color.Gray,
+                HeightRequest = 10,
+                WidthRequest = 160
+            };
+            
+            var boxView16 = new BoxView
+            {
+                HorizontalOptions = LayoutOptions.Start,
+                VerticalOptions = LayoutOptions.Start,
+                BackgroundColor = Color.Gray,
+                HeightRequest = 10,
+                WidthRequest = 175
+            };
+
+            grid.Children.Add(boxView1, 0, 0);
+            Grid.SetColumnSpan(boxView1, 2);
+
+            grid.Children.Add(boxView2, 0, 1);
+            Grid.SetColumnSpan(boxView2, 2);
+
+            grid.Children.Add(boxView3, 0, 2);
+            Grid.SetColumnSpan(boxView3, 2);
+
+            grid.Children.Add(boxView4, 0, 3);
+            Grid.SetColumnSpan(boxView4, 2);
+
+            grid.Children.Add(boxView5, 0, 5);
+            Grid.SetRowSpan(boxView5, 8);
+
+            grid.Children.Add(boxView6, 1, 5);
+
+            grid.Children.Add(boxView7, 1, 6);
+
+            grid.Children.Add(boxView8, 1, 7);
+
+            grid.Children.Add(boxView9, 1, 8);
+
+            grid.Children.Add(boxView10, 1, 9);
+
+            grid.Children.Add(boxView11, 1, 10);
+
+            grid.Children.Add(boxView12, 1, 11);
+
+            grid.Children.Add(boxView13, 1, 13);
+            Grid.SetRowSpan(boxView13, 3);
+
+            grid.Children.Add(boxView14, 0, 13);
+
+            grid.Children.Add(boxView5, 0, 14);
+
+            grid.Children.Add(boxView16, 0, 15);
+  
+            shimmer.CustomView = grid;
+            this.Content = shimmer;
+        }
+{% endhighlight %}
+
+{% endtabs %}
+
+## Shimmer view
+
+You can customize the `ShimmerView` using the `CustomView` property of `SfShimmer` and also you can change shape of the shimmer wave by using the property `ShapeType`.
```

Comment:
You can customize the `ShimmerView` using the `CustomView` property of `SfShimmer,` and also you can change the shape of the shimmer wave using the property the `ShapeType.`

Reply:
Contents modified